### DISCUSSION TITLE
How to delete commit year

### DISCUSSION THREAD

**Discussion by byuneunhye** [community/community#30745] [New to GitHub] [answered]
I joined GitHub in 2022. However, another year of the year was created.

**Comment by radar-blip** [comment, marked as answer]
If you read through this post. maybe you will find your answer: #27692

**Reply by byuneunhye** [reply]
Does that post mean that I can write an email to the github side?

**Reply by radar-blip** [reply]
Yes, read what is in the accepted answer, do the same thing.
Open support ticket and describe your problem, they will help you along.
https://support.github.com/contact/account?tags=dotcom-footer%2Chubberfy_account

**Reply by stawuah** [reply]
Please i created a repo and the code i had in that repo added me more commits dating back to 2019 , please i would want you to help me clear those old commits dating back to 2019 in exception of 2022 and 2023 , i would appreciate if this is being solved

**Comment by wonhyeongseo** [comment]
안녕하세요, [여기](https://support.github.com/contact/account?tags=dotcom-footer%2Chubberfy_account)에서 문의사항 접수하시면 깃헙에서 해결해주신다고 하네요. 위 질문에 쓰신대로만 쓰셔도 충분할 것 같아요. 그리고 아래를 하셔야 2008년에 실수로 넣은 커밋이 없어져요!

```
git rebase -i {커밋 해시값}
수정하려면 modify 선택
git commit --amend - no-edit --data =" 월 일 시:분:초 2008"
git rebase --continue
git push -f origin main 
```

Hello. Github will take care of it once you open a ticket [here](https://support.github.com/contact/account?tags=dotcom-footer%2Chubberfy_account). Describe the problem as you did above, and they will understand. Thank you.

**Reply by byuneunhye** [reply]
감사합니다!
